app/assets/stylesheets/experiment/table.scss

```diff
@@ -0,0 +1,85 @@
+// scss-lint:disable SelectorDepth NestingDepth IdSelector
+
+#experimentTable {
+  .experiment-table-container {
+    margin-top: 1em;
+    overflow-x: scroll;
+  }
+
+  .experiment-table {
+    display: grid;
+    grid-auto-rows: 3em 1px;
+    grid-template-columns: max-content repeat(calc(var(--columns-count)), minmax(max-content, auto)) max-content;
+    min-width: 100%;
+
+    .table-header-cell {
+      align-items: center;
+      background-color: $color-concrete;
+      border: 1px solid $color-white;
+      display: flex;
+      height: 3em;
+      padding: 0 .5em;
+      //position: sticky;
+      //position: -webkit-sticky;
+      //top: calc(var(--content-header-size) + var(--navbar-height));
```

Comment:
Why is this code commented on?

Reply:
We need to finish the toolbar before activating the sticky header. Without a toolbar then header will be In the wrong position.